test/tests/activitiesTest.js

```diff
@@ -111,8 +138,32 @@ module.exports = function (request, db) {
                 });
         });
 
-        it('should GET my sessions', function (done) {
-            request.get('/api/games/' + idGame + '/versions/' + idVersion + '/classes/' + idClass + '/sessions/my')
+        it('should GET my activities', function (done) {
+            request.get('/api/activities/my')
+                .expect(200)
+                .set('X-Gleaner-User', 'Teacher1')
+                .end(function (err, res) {
+                    should.not.exist(err);
+                    should(res).be.Object();
+                    should.equal(res.body.length, 1);
+                    done();
+                });
+        });
+
+        it('should GET my class activities', function (done) {
+            request.get('/api/classes/' + idClass + '/activities/my')
+                .expect(200)
+                .set('X-Gleaner-User', 'Teacher1')
+                .end(function (err, res) {
+                    should.not.exist(err);
+                    should(res).be.Object();
+                    should.equal(res.body.length, 1);
```

Comment:
Fixed in: https://github.com/e-ucm/rage-analytics-backend/pull/105/commits/539d5e5e799bf6faca746fa21db733b80c11dcc4